spec/stream_action_spec.rb

```diff
@@ -1,119 +0,0 @@
-require 'spec_helper'
-
-describe "Pubsubstub::StreamAction without EventMachine" do
```

Comment:
can you bring back some of the tests for the stream action plz. a simple "it works" will be enough I think